client/src/main/java/com/netflix/conductor/client/task/WorkflowTaskCoordinator.java

```diff
@@ -244,7 +240,7 @@ public Thread newThread(Runnable r) {
 				t.setName("workflow-worker-" + count.getAndIncrement());
```

Comment:
If you're going to stick with a Guava dependency, see com.google.common.util.concurrent.ThreadFactoryBuilder.